avocado/utils/disk.py

```diff
@@ -311,3 +312,101 @@ def get_disk_mountpoint(device):
             if dev == device:
                 return fs_dir
         return None
+
+
+def create_linux_raw_partition(disk_name, size=None, num_of_par=1):
+    """
+    Creates partitions using sfdisk command
+
+    :param disk_name: disk/device name
+    :type disk_name: str
+    :param size: size of partition
+    :type size: str
+    :param num_of_par: Number of partitions to be created
+    :type num_of_par: int
+
+    Returns list of created partitions
+    """
+    if not size:
+        size = get_size_of_disk(disk_name) / 1024 / 1024 / 1024
+        size = size / num_of_par
+        size = str(size) + "G"
+    partitions = [
+        "size= +" + size if val != 3 else "type=5" for val in range(0, num_of_par + 1)
+    ]
+    disk_partition_file = (
+        "/tmp/creat_partition" + process.run("date '+%d-%m-%y_%T'").stdout_text.strip()
+    )
+    if not os.path.isfile(disk_partition_file):
+        process.run("touch " + disk_partition_file)
+    for line in partitions:
+        genio.append_one_line(disk_partition_file, line)
+    part_output = process.getoutput("sfdisk " + disk_name + " < " + disk_partition_file)
+    for _ in range(60):
```

Comment:
range is hardcoded to 60 .. is that max partitions ?

Reply:
Hi Abdul! Used this to avoid sleep time.

Reply:
No Abdul! It's not max partitions. Just took a random number to wait for some time for the condition to be true

Reply:
```
flag = True
while flag:
    if <check something>:
         flag = False
         
```